src/Jsend/Response.php

```diff
@@ -0,0 +1,147 @@
+<?php
+declare(strict_types=1);
+
+namespace Assertis\Util\Jsend;
+
+use DateTime;
+use JsonSerializable;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+/**
+ * @author Maciej Romanski <maciej.romanski@assertis.co.uk>
+ */
+class Response extends JsonResponse
+{
+    /**
+     * @param ResponseData $data
+     * @param int $status
+     * @param array $headers
+     */
+    public function __construct(ResponseData $data, int $status = self::HTTP_OK, array $headers = [])
+    {
+        parent::__construct($data, $status, $headers);
+    }
+
+    /**
+     * @param int $seconds
+     * @return Response
+     */
+    public function cacheFor(int $seconds): Response
+    {
+        $this->setSharedMaxAge($seconds);
+        $this->setMaxAge($seconds);
+        $this->setExpires(new DateTime("+{$seconds} seconds"));
+
+        return $this;
+    }
+
+    /**
+     * Return success response
+     *
+     * @param array|JsonSerializable|null $data
+     * @param int $status
+     * @param array $headers
+     * @param array $links
+     * @return Response
+     */
+    public static function success($data, int $status = self::HTTP_OK, array $headers = [], array $links = []): Response
+    {
+        return new self(
+            new ResponseData(
+                new StatusEnum(StatusEnum::SUCCESS),
+                $data,
+                $links,
+                null,
+                null
+            ),
+            $status,
+            $headers
+        );
+    }
+
+    /**
+     * @param string $uri
+     * @param array|JsonSerializable|null $data
+     * @param array $headers
+     * @return Response
+     */
+    public static function created(string $uri, $data = null, array $headers = []): Response
+    {
+        $data['uri'] = $uri;
+        $headers['Location'] = $uri;
+
+        return self::success($data, self::HTTP_CREATED, $headers);
+    }
+
+    /**
+     * @param string $message
+     * @param array $headers
+     * @return Response
+     */
+    public static function notFound(string $message = null, array $headers = [])
+    {
+        $status = new StatusEnum(StatusEnum::FAIL);
+
+        return new self(new ResponseData($status, null, null, $message, null), self::HTTP_NOT_FOUND, $headers);
```

Comment:
keep the same formatting of ResponseData pls https://github.com/assertis/util-lib/pull/31/files#diff-b571bc5e48c00e22444a56343060d192R49